system/Email/Email.php

```diff
@@ -279,6 +279,13 @@ class Email
      */
     protected $SMTPAuth = false;
 
+    /**
+     * Which SMTP authentication method to use: login, plain
+     *
+     * @var string
+     */
+    protected $SMTPAuthMethod = 'login';
```

Comment:
as this is a new property I think it is safe to declare the string type natively?
```suggestion
    protected string $SMTPAuthMethod = 'login';
```

Reply:
Good point